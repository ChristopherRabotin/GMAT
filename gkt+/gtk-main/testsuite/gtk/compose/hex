<Multi_key> <s> <e> <q> : "\x23fe\X23F3"
