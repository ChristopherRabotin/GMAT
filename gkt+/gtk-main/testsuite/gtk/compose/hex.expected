# n_sequences: 1
# max_seq_len: 4
# n_index_size: 1
# data_size: 9
# n_chars: 7
<Uff20> <U73> <U65> <U71> : "⏾⏳"
